### PR TITLE
Mention Minikube in the Kubectl configuration doc

### LINKED ISSUE #2863: Issue with /docs/user-guide/prereqs/
**This is a...** 

- [ ] Feature Request
- [ x] Bug Report

**Problem:**
The Minikube installation (https://kubernetes.io/docs/getting-started-guides/minikube/) has a per-requisite of Installing and Setting up kubectl. So it's natural for users to follow all of the 'prereqs' steps as mentioned in the instructions, however the verification of Kubectl (i.e. $ kubectl cluster-info) as explained under 'Configuring kubectl/Making sure you're ready' won't produce an expected output unless Minikube was installed in user's system. The new users may not have any idea of why it's not working.

**Proposed Solution:**
At least mention Minikube along with the mention of kube-up.sh on creating cluster. This will give convey message to first time users of the Minikube that they need to make sure Minikube is installed before running this step of $ kubectl cluster-info

**Page to Update:**
http://kubernetes.io/...

### PR DESCRIPTION
**Pull request by spzala**
The Minikube installation (https://kubernetes.io/docs/getting-started-guides/minikube/) has a per-requisite of Installing and Setting up kubectl. So it's natural for users to follow all of the 'prereqs' steps as mentioned in the instructions, however the verification of Kubectl (i.e. $ kubectl cluster-info) as explained under 'Configuring kubectl/Making sure you're ready' won't produce an expected output unless Minikube was installed in user's system. The new users may not have any idea of why it's not working.

This PR changes add a reference of Minikube along with kube-up.sh. This will convey message to first time users of the Minikube that they should make sure Minikube is installed before running this step (i.e. $ kubectl cluster-info)

fixes #2863

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2864)

### CODE REVIEW

**Comment by spzala** [comment]
@mikedanese Hi Mike, I was not up-to-date with master :( and the content of the file I made changes was moved to a new file. Fixing this with git, as you can notice, created multiple commits. Please let me know if that doesn't sound good? In that case, I guess the best alternative for me is to close the PR request here and address changes with new PR. Thanks!

File: docs/tasks/kubectl/install.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ sufficiently up-to-date using `kubectl version`.
 
 ## Configuring kubectl
 
-In order for kubectl to find and access the Kubernetes cluster, it needs a [kubeconfig file](/docs/concepts/cluster-administration/authenticate-across-clusters-kubeconfig/), which is created automatically when creating a cluster using kube-up.sh (see the [getting started guides](/docs/getting-started-guides/) for more about creating clusters). If you need access to a cluster you didn't create, see the [Sharing Cluster Access document](/docs/tasks/administer-cluster/share-configuration/).
+In order for kubectl to find and access the Kubernetes cluster, it needs a [kubeconfig file](/docs/concepts/cluster-administration/authenticate-across-clusters-kubeconfig/), which is created automatically when creating a cluster using kube-up.sh or upon successful installation of Minikube (see the [getting started guides](/docs/getting-started-guides/) for more about creating clusters). If you need access to a cluster you didn't create, see the [Sharing Cluster Access document](/docs/tasks/administer-cluster/share-configuration/).
```
Comment (mikedanese) [inline docs/tasks/kubectl/install.md]: should say successful deployment of minikube cluster, not successful installation.

**Comment by mikedanese** [comment]
This needs to be one commit.

File: docs/tasks/kubectl/install.md (hunk 1 above)
Comment (spzala) [inline docs/tasks/kubectl/install.md]: @mikedanese sure, that sounds good to me! Thanks!

**Comment by spzala** [comment]
@mikedanese thanks Mike, that will be great!

**Comment by spzala** [comment]
@mikedanese made changes per your suggestion but added new commits :-). A k8s newbie question - what's a recommended way to update changes for reviewer(s) comments? Per some official git blog recommended way is to create new commit (i.e. leave the original commit as such and create new one every time you make changes), some folks prefer use of --amend and some prefer reset? I guess all are valid and it can be a personal preference but if there is a best practice, would love to know it.  Thanks!!

**Pull request merged**